src/UserStory.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { Image } from "@chakra-ui/react"
+import { Button, ButtonGroup, Stack } from "@chakra-ui/react"
+export default class UserStory extends React.Component {
+    render() {
+        return(
+            <Button colorScheme="teal" size="md" variant="ghost">
+                <Image src={this.props.src} alt="user69"boxSize="60px" onClick={this.props.onClick}/>
```

Comment:
Perhaps it would be better to pass down the value of `alt` as a prop.
```suggestion
                <Image src={this.props.src} alt={this.props.alt} boxSize="60px" onClick={this.props.onClick}/>
```